packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -1695,11 +1695,7 @@ class Package {
       mod.emit(code, context);
       code.closeFile(filename);
 
-      for (const script of mod.emitBinScripts(code)) {
-        if (scripts.indexOf(script) < 0) {
-          scripts.push(script);
-        }
-      }
+      Array.prototype.push.apply(scripts, mod.emitBinScripts(code));
```

Comment:
Why not `scripts.push(...mod.emitBinScripts(code))`?

Reply:
I just not very familiar with typescript so had no idea about '...' operator.